### DISCUSSION TITLE
Um SOS tech :)

### DISCUSSION THREAD

**Discussion by eianasofia** [frontendbr/forum#2417] [Discussão]
Oi gente, tudo bem?
Estou começando a me aventurar no mundo da tecnologia para realizar transição de carreira e confesso que estou um pouco perdida no meio de tantas opções de cursos gratuitos e pagos, além de algumas dúvidas sobre a carreira terem surgido. Poderiam me ajudar com algumas dúvidas (enumeradas para ser mais fácil responder o que quiserem)? Vamos lá:

1. É mesmo necessário ter uma graduação ou pelo menos curso técnico? Ou cursos livres funcionam igualmente para o mercado? 
2. Alguém tem experiência com a CoderHouse pra me falar como foi o curso e a experiência com eles?
3. Na opinião de vocês que já estão inseridos na área, se fossem começar a estudar agora, que ramo evitariam porque acreditam que ficará mais rapidamente ultrapassado?

Foi mal se fiz alguma pergunta muito besta, mas começo é isso mesmo né haha
Desde já muito obrigada!

**Comment by g3rley** [comment]
Olá Sofia, tudo bem?

Suas dúvidas são muito pertinentes e merecem sim um bom esclarecimento. Com quase 10 anos no mercado tenho percebido alguns padrões que as empresas tem buscado. O entendimento desses padrões pode ajuda a sanar essas dúvidas:

1) É mesmo necessário ter uma graduação ou pelo menos curso técnico? Ou cursos livres funcionam igualmente para o mercado?
R= Não é necessário ter uma graduação ou um curso técnico. Em uma régua de contratação, portfólios e o engajamento desses projetos nos principais veículos que esses contratantes se encontram são muito mais importantes. Contudo, em vagas que se esperam um grande número de candidatos, pessoas que não possuem um curso superior acabam não sendo selecionadas em um primeiro filtro. Nada que um tecnólogo de baixa duração não resolva esse tipo de circunstância. Mas isso não é regra! É muito comum pessoas que ainda não possuem um ensino superior já estarem empregadas, por vezes, em cargos de certa senioridade.

2) Alguém tem experiência com a CoderHouse pra me falar como foi o curso e a experiência com eles?
R= Como não tenho experiência, vou deixar para um coleguinha responder essa pergunta em um momento mais oportuno.

3) Na opinião de vocês que já estão inseridos na área, se fossem começar a estudar agora, que ramo evitariam porque acreditam que ficará mais rapidamente ultrapassado?
R= Se eu fosse começar a estudar agora eu evitaria, com toda a tranquilidade do mudo, uma posição em um órgão público. Especialmente se for exercer alguma atividade em uma secretaria apenas sendo um profissional terceirizado. Normalmente as ferramentas são muito limitadas e há uma dificuldade muito grande em se trabalhar com novidades do mercado. Por uma questão política, é muito fácil a urgência das atividades acabarem perdendo o controle. Esse tipo de experiência para um iniciante na área pode ser extremamente estressante.

Espero que tenha esclarecido algumas dúvidas! 😊 😀 😁

**Reply by eianasofia** [reply]
Muito obrigada pela resposta e pelo tempo, Gérley! Muito bom ler sobre a experiência de alguém com tanto tempo de mercado. Eu sou graduada em Comunicação e estou considerando um técnico agora. Vi que você trabalha com data, a princípio me interesso pela área também, vamos ver a medida que for avançando nos estudos... Animada! Obrigada mais uma vez :)
